api/Navigator.json

```diff
@@ -3744,7 +3744,8 @@
             "edge": "mirror",
             "firefox": [
               {
-                "version_added": "preview"
+                "version_added": "preview",
+                "notes": "API access is gated by installation of a site permission add-on (user prompt), secure context, and <a href='https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/midi'>Permission Policy: <code>midi</code></a>."
```

Comment:
The preview version contains specific information about the site permission add on and feature policy gating. 
We will need to link to docs on the site permission when Firefox has written them.